pkg/scheduler/actions/allocate/allocate.go

```diff
@@ -222,9 +222,9 @@ func (alloc *allocateAction) Execute(ssn *framework.Session) {
 				if task.InitResreq.LessEqual(node.Releasing) {
 					glog.V(3).Infof("Pipelining Task <%v/%v> to node <%v> for <%v> on <%v>",
 						task.Namespace, task.Name, node.Name, task.InitResreq, node.Releasing)
-					if err := stmt.Pipeline(task, node.Name); err != nil {
-						glog.Errorf("Failed to pipeline Task %v on %v",
-							task.UID, node.Name)
+					if err := ssn.Pipeline(task, node.Name); err != nil {
```

Comment:
any more info on why change it from `stmt` to `ssn` ?

Reply:
If `reclaim` action is enabled after `allocate` action. The cluter situation is as below

| serial   | node name | resource  |
|----------|------------|---------|
| 1        | node1    | 4c8g       |
| 2        | node2    | 4c8g  |

| serial   | queue name | weight  | quota| status
|----------|------------|---------|---|---- |
| 1        | default    | 1       | 0.8c 1.5M | overused |
| 2        | queue1     | 100000  | 7c 8g| active |

| serial   | job name | pods number  |minA | queue|status |
|----------|------------|---------|---|---|---|
| 1        | joba    | 7       | 1|default | all running |
| 2        | jobb     | 7  | 7| queue1 | all pending|

There are two jobs in the cluster, joba and jobb. Joba was placed in `default` queue. Jobb was placed in `queue1` queue. Joba  has 7pods Running. Jobb has 7pods pending. `default` queue was overused. Pods in `queue1` will try to reclaim resource from `defualt` queue. 

* In `reclaim` action, `podb-1` in `jobb` evicted pod `poda-1` in `joba`, `poda-1` was originally at node `node1`,  the scheduling loop ends.

* In the next scheduling loop. In `allocate` action,  `podb-1` want to be pipelined to node `node1`, but gang-restriction of `jobb`  was not meet, the pipeline action will be discard. In `allocate` action no pod was pipelined, though there are releasing resources in cluster.

* In `relcaim` action of this scheduling loop, `podb-1` in `jobb` will try to evicted other pods in `joba`.

* Finally, `podb-1` will evicted 6 pods from `joba`.

* So if there are releasing resources in cluster, pod who has the higher priority, may should be `pipelined` to the node, regardless of whether  the gang restriction of job of the pod was meet, in case that, the pod will evict other pods in subsequent actions.